image_provisioner/playbooks/roles/os-kickstart/tasks/main.yaml

```diff
@@ -1,69 +1,88 @@
 ---
-- name: yum update
-  yum: name=* state=latest
+- block:
+  - name: yum update
+    yum: name=* state=latest
 
-- name: install packages
-  yum: name={{ item }} state=latest
-  with_items:
-    - ansible
-    - bind-utils
-    - blktrace
-    - ethtool
-    - findutils
-    - git
-    - gnuplot
-    - golang
-    - httpd-tools
-    - hwloc
-    - iotop
-    - iptables-services
-    - kernel
-    - ltrace
-    - mailx
-    - maven
-    - netsniff-ng
-    - net-tools
-    - ntp
-    - ntpdate
-    - numactl
-    - pciutils
-    - perf
-    - pbench-agent
-    - pbench-sysstat
-    - python-docker-py
-    - python-flask
-    - python-pip
-    - python-rbd
-    - python2-boto3
-    - powertop
-    - psmisc
-    - rpm-build
-    - screen
-    - sos
-    - strace
-    - tar
-    - tcpdump
-    - tmux
-    - vim-enhanced
-    - xauth
-    - wget
-    - yum-utils
-    - rpmdevtools
-    - iperf3
+  - name: install packages
+    yum: name={{ item }} state=latest
+    with_items:
+      - ansible
+      - bind-utils
+      - blktrace
+      - ethtool
+      - findutils
+      - git
+      - gnuplot
+      - golang
+      - httpd-tools
+      - hwloc
+      - iotop
+      - iptables-services
+      - kernel
+      - ltrace
+      - mailx
+      - maven
+      - netsniff-ng
+      - net-tools
+      - ntp
+      - ntpdate
+      - numactl
+      - pciutils
+      - perf
+      - pbench-agent
+      - pbench-sysstat
+      - python-docker-py
+      - python-flask
+      - python-pip
+      - python-rbd
+      - python2-boto3
+      - powertop
+      - psmisc
+      - rpm-build
+      - screen
+      - sos
+      - strace
+      - tar
+      - tcpdump
+      - tmux
+      - vim-enhanced
+      - xauth
+      - wget
+      - yum-utils
+      - rpmdevtools
+      - iperf3
+      - docker
 
-- name: install pbench internal
-  yum: name=pbench-agent-internal state=latest
-  when: internal_image | bool
+  - name: install pbench internal
+    yum: 
+      name: pbench-agent-internal 
+      state: latest
+    when: internal_image | bool
 
-- name: install persistent storage clients
-  yum: name={{ item }} state=latest
-  with_items:
-    - ceph-common
-    - glusterfs-fuse
-    - iscsi-initiator-utils
+  - name: update pbench-agent to get the latest changes
+    yum: 
+      name: pbench-agent 
+      enablerepo: ndokos-pbench-interim
+      state: latest
+
+  - name: install persistent storage clients
```

Comment:
Not sure if we want another package installation step? Maybe add these to common install above.